### ISSUE TITLE
Password for p12 created using fastlane match

### ISSUE THREAD

**Issue by skdev03** [fastlane/fastlane#15564]
### Question Checklist

- fastlane 2.134.0

### Question Subject

- What is the password for a p12 created using fastlane match?

#### Question Description
- I tried to open a p12 file using different kinds of passwords (passphrase, empty password, password for the repo, keychain password), but nothing is working. What will be the password for that p12 file?

**Comment by skdev03** [comment]
Found out more about how to decrypt the encrypted .cer and .p12 files, but the manual decryption is giving a "bad encrypt" error.

Command:
openssl aes-256-cbc -k <password> -in RBXBU2929L.p12 -out RBXBU2929L.new.p12 -a -d

I entered the passphrase as the password. Is that correct?
I'm getting this error after executing the above command

bad decrypt
4354399724:error:06FFF064:digital envelope routines:CRYPTO_internal:bad decrypt:/BuildRoot/Library/Caches/com.apple.xbs/Sources/libressl/libressl-47.11.1/libressl-2.8/crypto/evp/evp_enc.c:521:

Does anyone know how to resolve this error?
